pkgs/CAFxX/mgo/registry.yaml

```diff
@@ -0,0 +1,8 @@
+packages:
+  - type: go_install
+    repo_owner: CAFxX
+    repo_name: mgo
+    description: Build and bundle multiple GOAMD64 variants in a single executable
+    supported_envs:
+      - darwin
+      - linux
```

Comment:
I'm not familiar with this tool, but this tool doesn't work on Windows?

Reply:
```console
$ mgo --help
INFO[0000] Installing a Go tool                          aqua_version=2.21.0 env=darwin/arm64 exe_name=mgo go_package_path=github.com/CAFxX/mgo@v0.2.12 gobin=/Users/shunsukesuzuki/.local/share/aquaproj-aqua/pkgs/go_install/github.com/CAFxX/mgo/v0.2.12/bin package_name=CAFxX/mgo package_version=v0.2.12 program=aqua registry=standard
go: downloading github.com/CAFxX/mgo v0.2.12
go: downloading golang.org/x/sync v0.2.0
go: downloading golang.org/x/mod v0.10.0
GOOS="darwin" is not supported
```

Oh? `darwin` isn't supported?

Reply:
> Currently only GOOS=linux and GOARCH=amd64 are supported, and only in buildmodes that produce executables (not archives, plugins, or libraries)

https://github.com/CAFxX/mgo/blob/8e4a2e5f301ba679868b70fdc27ae7cbd47b806a/README.md?plain=1#L50-L51

Reply:
Yes, but should I limit it to `linux/amd64` only?

![CleanShot 2023-12-13 at 07 51 40@2x](https://github.com/aquaproj/aqua-registry/assets/121048/19689be5-a5dd-4833-a5e1-4f18789b8e9e)
https://github.com/CAFxX/mgo#notes

Reply:
Yeah, I think so.

Reply:
fixed `supported_envs`